src/exchange/information/information.html

```diff
@@ -4,257 +4,205 @@
-                                                'oui-progress_error': ctrl.exchange.diskPercent > 80
-                                            }">
-                                            <div class="oui-progress__bar" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="{{ctrl.exchange.diskPercent}}"
-                                                 data-ng-class="{
-                                                    'oui-progress__bar_success': ctrl.exchange.diskPercent < 51,
-                                                    'oui-progress__bar_warning': ctrl.exchange.diskPercent > 50 && ctrl.exchange.diskPercent < 81,
-                                                    'oui-progress__bar_error': ctrl.exchange.diskPercent > 80
-                                                }"
-                                                 data-ng-style="{'width': ctrl.exchange.diskPercent + '%'}">
-                                                <span class="oui-progress__label"
-                                                      data-ng-bind="ctrl.exchange.currentDiskUsage.value + ' ' + ('unit_size_'+ ctrl.exchange.currentDiskUsage.unit | translate) + ' / ' + ctrl.exchange.totalDiskSize.value + ' ' + ('unit_size_'+ ctrl.exchange.totalDiskSize.unit | translate)"></span>
-                                            </div>
-                                        </div>
-                                    </div>
-                                    <oui-action-menu data-compact 
-                                                     data-align="end"
-                                                     data-disabled="ctrl.exchangeServiceInfrastructure.is25g()"
-                                                     data-ng-if="ctrl.displayOrderDiskSpace()">
-                                        <oui-action-menu-item data-on-click="ctrl.orderDiskSpace()">
-                                            <span data-translate="exchange_action_order_space_disk"></span>     
-                                        </oui-action-menu-item>
-                                    </oui-action-menu>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'exchange_tab_INFORMATIONS_stats' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_domains' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.domainsNumber"></span>
+                            </oui-tile-description>
+                        </oui-tile-definition>
+                        <oui-tile-definition data-term="{{:: 'exchange_dashboard_accounts' | translate}}">
+                            <oui-tile-description>
+                                <span data-ng-bind="ctrl.exchange.accountsNumber"></span>
```

Comment:
Refers to comment : https://github.com/ovh-ux/ovh-module-exchange/pull/181/files#r229202742